README.md

```diff
@@ -1,4 +1,34 @@
 # IFS
 Scrum Master: Jan Greger
+
 Build Master: Magnus
+
 Product Owner: Simon MC
+
+## Installation
+    $ pip install -r requirements.txt
+
+## Configuration
+Default settings file:
+
+    ifs/config.py
+
+Optional settings files:
+
+    1.  path from environment variable 'APPSTATS_SETTINGS'
+    2.  /etc/appstats.cfg
+    3.  ~/.appstats.cfg
```

Comment:
These do not match the names/paths used in the code at ifs/__init__.py line 11 to 13. They should, right? Also, why do you use the name 'appstats' or 'librarian'?